ironfish/src/primitives/note.test.ts

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { createNodeTest, useAccountFixture, useMinerBlockFixture } from '../testUtilities'
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+describe('Note', () => {
+  const nodeTest = createNodeTest()
+
+  it('check if a transaction is not a miners fee', async () => {
+    const nodeA = nodeTest.node
+
+    // Create an account A
+    const accountA = await useAccountFixture(nodeA.wallet, () =>
+      nodeA.wallet.createAccount('testA'),
+    )
+    const accountB = await useAccountFixture(nodeA.wallet, () =>
+      nodeA.wallet.createAccount('testB'),
+    )
+
+    // Create a block with a miner's fee
+    const block1 = await useMinerBlockFixture(nodeA.chain, 2, accountA)
+    await nodeA.chain.addBlock(block1)
+    await nodeA.wallet.updateHead()
+
+    const transaction = await nodeA.wallet.createTransaction(
```

Comment:
If this test needs these and isn't going to be refactored away, use a transaction fixture here - transactions are the heaviest part of most code. `useTxFixture` probably? Been a bit since I've worked with them